adserver/views.py

```diff
@@ -1970,7 +1986,18 @@ class StaffRegionReportView(AllReportMixin, BaseReportView):
 
     """An region report for all publishers."""
 
-    fieldnames = ["index", "views", "clicks", "ctr"]
+    export_view = "staff_region_report_export"
```

Comment:
This is causing the test failure although that check should just be removed. The check was ensuring that "CSV Export" didn't appear since it wasn't implemented.